Missing type definitions since 1.3.0

## Expected behavior

No TypeScript errors.

## Current behavior

TypeScript outputs the error:

```
Cannot find module '@researchgate/react-intersection-observer' or its corresponding type declarations.ts(2307)
```

<img width="539" alt="image" src="https://user-images.githubusercontent.com/473530/89021902-32be3680-d35c-11ea-92a6-c8e0560e875b.png">

## Steps to reproduce

See https://codesandbox.io/s/nostalgic-shadow-qz6gh?file=/src/App.tsx

## Context (environment)

* **Version**: 1.3.0
* **Platform**: `Darwin F200420NT01.local 19.6.0 Darwin Kernel Version 19.6.0: Sun Jul  5 00:43:10 PDT 2020; root:xnu-6153.141.1~9/RELEASE_X86_64 x86_64`

--- a/package.json
+++ b/package.json
@@ -56,8 +56,8 @@
   ],
   "license": "MIT",
   "main": "lib/js/index.js",
-  "module": "lib/es/index.js",
-  "types": "typings/index.d.ts",
+  "module": "lib/es/src/index.js",
+  "types": "typings/src/index.d.ts",
   "peerDependencies": {
     "react": "^16.3.2",
     "react-dom": "^16.3.2"

